zerver/openapi/openapi.py

```diff
@@ -13,21 +14,63 @@
 EXCLUDE_PROPERTIES = {
     '/register': {
         'post': {
-            '200': ['max_message_id', 'realm_emoji'],
+            '200': ['max_message_id', 'realm_emoji', 'pointer'],
```

Comment:
In theory, the `register` response should have like 50 more items listed here; this may be because of details in how we call it in `test_events` that avoid actually using the API endpoint.  May want to rethink that approach, but that's certainly for follow-up work.